src/transformStateWithClones.js

```diff
@@ -6,8 +6,43 @@
  *
  * @return {Object[]}
  */
+
+const add = 'addProperties';
+const remove = 'removeProperties';
+const clear = 'clear';
```

Comment:
I'd consider placing those variables inside the function maybe? Just for clarity in this case